.env.example

```diff
@@ -1,27 +1,31 @@
 # POSTGRES
-DB_USER=userhere
-DB_PASSWORD=passhere
-DB_NAME=dbnamehere
-DB_PORT=dbporthere
+DB_HOST=localhost
+DB_PORT=5432
+DB_USER=user
+DB_PASSWORD=password
+DB_NAME=localdb
 POSTGRES_DATA=./db_data
 
 # HAPI
-VIRTUAL_HOST=virtualhosthere
-VIRTUAL_PORT=virtualporthere
+VIRTUAL_HOST=0.0.0.0
+VIRTUAL_PORT=9090
 EOS_API_ENDPOINT=https://jungle.eosio.cr
 PROXY_INFO_CONTRACT_CODE=proxyaccount
 PROXY_INFO_CONTRACT_SCOPE=proxyaccount
 
 # HASURA
-HASURA_GRAPHQL_DATABASE_URL=dburlhere
-HASURA_GRAPHQL_ADMIN_SECRET=hasurasecrethere
-HASURA_GRAPHQL_UNAUTHORIZED_ROLE=rolehere
+HASURA_GRAPHQL_DATABASE_URL=postgres://user:password@eosrate_postgres:5432/localdb
+HASURA_GRAPHQL_ADMIN_SECRET=hasurasecret
+HASURA_GRAPHQL_UNAUTHORIZED_ROLE=guest
+HASURA_GRAPHQL_ACTION_BASE_URL=http://hapi:9090
 
 # WEBAPP
 REACT_APP_GRAPHQL_HTTP_URL=http://localhost:8080/v1/graphql
+REACT_APP_API_URL=https://jungle.eosio.cr
 REACT_APP_GRAPHQL_WS_URL=ws://localhost:8080/v1/graphql
 REACT_APP_EOS_API_URL=https://jungle.eosio.cr
 REACT_APP_EOS_API_HOST=jungle.eosio.cr
 REACT_APP_EOS_API_PORT=443
 REACT_APP_EOS_API_PROTOCOL=https
-REACT_APP_EOS_CHAIN_ID=2a02a0053e5a8cf73a56ba0fda11e4d92e0238a4a2aa74fccf46d5a910746840
\ No newline at end of file
+REACT_APP_EOS_CHAIN_ID=2a02a0053e5a8cf73a56ba0fda11e4d92e0238a4a2aa74fccf46d5a910746840
+REACT_APP_BLOCK_EXPLORER=blockexplorer
```

Comment:
set this to `REACT_APP_BLOCK_EXPLORER: https://jungle3.bloks.io`

Reply:
you can use this file as reference for the local .env.example values 

https://github.com/eoscostarica/eos-rate/blob/staging/.github/workflows/push-staging-environment.yaml